File: MedContactCore/DataTransferObjects/RoleDto.cs
using MedContactDb.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MedContactCore.DataTransferObjects
{
    public class RoleDto 
    {
        public Guid Id { get; set; }
        public string? Name { get; set; }
        public List<User>? Users { get; set; }
        public List<DoctorDataDto>? DoctorDatas { get; set; }
        public List<CustomerDataDto>? CustomerDatas { get; set; }
        public List<AcsDataDto>? AcsDatas { get; set; }
       

    }
}

File: medcontact-angular/medcontact/src/app/models/roledto.ts
import { Acsdatadto } from "./acsdatadto";
import { Customerdatadto } from "./customerdatadto";
import { Doctordatadto } from "./doctordatadto";
import { Userdto } from "./userdto";

export class Roledto {
  id?: string;
  name?: string;
  isSelected?: boolean;
  users?: Array<Userdto>;
  doctorDatas?: Array<Doctordatadto>;
  customerData?: Array<Customerdatadto>;
  acsDatas?: Array<Acsdatadto>;
}
